bin/next-start

```diff
@@ -48,13 +57,13 @@ if (!existsSync(resolve(dir, '.next', 'BUILD_ID'))) {
 }
 
 srv.start(argv.port, argv.hostname)
-.then(() => {
+.then((server) => {
+  const {address, port} = server.address()
   if (!process.env.NOW) {
-    console.log(`> Ready on http://${argv.hostname && typeof argv.hostname !== 'boolean' ? argv.hostname : 'localhost'}:${argv.port}`)
+    console.log(`> Ready on http://${address}:${port}`)
```

Comment:
I'm not sure we we'd like to use `server.address()` here since it's a resolved IP instead of hostname. I feel hostname would be better to display.